R/CorrelationMethods.R

```diff
@@ -651,3 +651,28 @@ yaxs = par("yaxs"), ...)
         points(x[sel, ], pch = pch, cex = cex, col = col)
     }
 }
+
+# A simple function to calculate correlation matrix without plotting it
```

Comment:
You need a proper documentation page otherwise R CMD check issues a WARNING that fails GitHub's CI tests.